SeeSharp/Geometry/MeshLoadException.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+namespace SeeSharp.Geometry {
+    public class MeshLoadException : Exception {
+        public string Path { get; }
+        public MeshLoadException(string message, string path) : base(message) {
```

Comment:
Might be nice to make `Path` a part of the final exception message. Maybe like: `message + $" (Path)"`